### ISSUE TITLE
A lot of non-standard descriptions

### ISSUE THREAD

**Issue by ValWood** [pombase/pombase-chado#586]
Not sure what this means, if it menas what I think it means it can be standardized as a multiple AA alteration:
amino acid 73 deleted,Y71A
amino acid 73 deleted,Y72A
amino acid 73 deleted,YY71DD
amino acid 73 deleted,YY71GG
amino acid 73 deleted,YY71KK

we have a standard way to do these
2137 OF 2181 5' UTR deleted
58KRRKR62 substituted with NEHG
71KKRKK75 substituted with NEHG
DRE1 and DRE2 AGGTG/A->CTTCA/G

GVPHE inserted after E94
GVPHS inserted after S7
GVPHY inserted after Y124
GVPLA inserted after A195
GVPLD inserted after D331

etc

lots.

I suggest first step, Kim report to logs EVERYTHING which does not match the syntax for the description.
Once we have these we need to see if we can convert to standard format, and if not, describe as "OTHER"

**Comment by mah11** [comment]
> amino acid 73 deleted,Y71A [etc.]

At present we don't have a non-"other" way to indicate both a deletion and a substitution in the same allele.

> 2137 OF 2181 5' UTR deleted

Yes, this looks like deletion of nt between -2221 and -5 inclusive (if I've got the arithmetic right).

> 58KRRKR62 substituted with NEHG
> 71KKRKK75 substituted with NEHG

OK, I think these could be done as KRRKR58NEHG and KKRKK71NEHG, if we're happy to use this syntax when 5 amino acids are replaced by 4.

> DRE1 and DRE2 AGGTG/A->CTTCA/G

I also looked up the others from that session:

AG2CG(DRE1 & DRE2 T->C) is T-239C,T-190C
ATGTG(DRE1 & DRE2 G->T) is G-237T,G-188T
AT2CG(DRE1 & DRE2 GGT->TTC) is GGT-237TTC,GGT-188TTC
CT2CA(DRE1 and DRE2 AGGTG/A->CTTCA/G) is AGGTG-236CTTCA,AGGTA-187CTTCG

> GVPHE inserted after E94 [etc.]

At present we don't have a systematic description syntax for insertions. They're an obvious candidate for a new allele type, though.
